README.md

```diff
@@ -30,6 +30,17 @@ make demo
 The build script will then download data file for Doom automatically. SDL2 based window
 should appear when Doom is loaded and executed.
 
+### Run riscv-compliance test
+Once the submodule `riscv-compliance` is pulled, run all the available compliance test via command:
```

Comment:
In `Makefile`, you can add submodule specific operations to pull automatically. See https://github.com/sysprog21/jitboy/blob/master/Makefile